opsmngr/automation_config.go

```diff
@@ -231,6 +231,12 @@ type MongoDBUser struct {
 	Username                   string                      `json:"user"` //nolint:tagliatelle // Username is a better name than just user
 }
 
+// MongoDBDeletedUser user to delete from DBs list.
+type MongoDBDeletedUser struct {
+	Username  string   `json:"user"` //nolint:tagliatelle // used as in the API
+	Databases []string `json:"dbs"`  //nolint:tagliatelle // used as in the API
```

Comment:
let's follow tagliatelle
```suggestion
	User  string   `json:"user"` //nolint:tagliatelle // used as in the API
	DBs []string `json:"dbs"`  //nolint:tagliatelle // used as in the API
```

Reply:
had to keep //nolint for dbs as it's not accepted this way and needs to be dBs